[2024-09-21T22:12:29.220+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_ingestion.trigger_stage_processing manual__2024-09-21T22:09:52.502170+00:00 [queued]>
[2024-09-21T22:12:29.231+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_ingestion.trigger_stage_processing manual__2024-09-21T22:09:52.502170+00:00 [queued]>
[2024-09-21T22:12:29.231+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 0
[2024-09-21T22:12:29.244+0000] {taskinstance.py:1382} INFO - Executing <Task(TriggerDagRunOperator): trigger_stage_processing> on 2024-09-21 22:09:52.502170+00:00
[2024-09-21T22:12:29.249+0000] {standard_task_runner.py:57} INFO - Started process 1888 to run task
[2024-09-21T22:12:29.251+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'raw_ingestion', 'trigger_stage_processing', 'manual__2024-09-21T22:09:52.502170+00:00', '--job-id', '416', '--raw', '--subdir', 'DAGS_FOLDER/raw_ingestion.py', '--cfg-path', '/tmp/tmpg4uhrqe4']
[2024-09-21T22:12:29.252+0000] {standard_task_runner.py:85} INFO - Job 416: Subtask trigger_stage_processing
[2024-09-21T22:12:29.357+0000] {task_command.py:416} INFO - Running <TaskInstance: raw_ingestion.trigger_stage_processing manual__2024-09-21T22:09:52.502170+00:00 [running]> on host airflow-worker
[2024-09-21T22:12:29.525+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_ingestion' AIRFLOW_CTX_TASK_ID='trigger_stage_processing' AIRFLOW_CTX_EXECUTION_DATE='2024-09-21T22:09:52.502170+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-21T22:09:52.502170+00:00'
[2024-09-21T22:12:29.602+0000] {trigger_dagrun.py:173} INFO - Clearing stage_processing on 2024-09-21T00:00:00+00:00
[2024-09-21T22:12:29.699+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/trigger_dagrun.py", line 163, in execute
    dag_run = trigger_dag(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/api/common/trigger_dag.py", line 122, in trigger_dag
    triggers = _trigger_dag(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/api/common/trigger_dag.py", line 78, in _trigger_dag
    raise DagRunAlreadyExists(dag_run=dag_run, execution_date=execution_date, run_id=run_id)
airflow.exceptions.DagRunAlreadyExists: A DAG Run already exists for DAG stage_processing at 2024-09-21T00:00:00+00:00 with run id manual__2024-09-21T00:00:00+00:00

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/trigger_dagrun.py", line 183, in execute
    dag.clear(start_date=parsed_execution_date, end_date=parsed_execution_date)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/session.py", line 79, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dag.py", line 2250, in clear
    clear_task_instances(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 260, in clear_task_instances
    ti.max_tries = ti.try_number + task_retries - 1
TypeError: unsupported operand type(s) for +: 'int' and 'NoneType'
[2024-09-21T22:12:29.727+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=raw_ingestion, task_id=trigger_stage_processing, execution_date=20240921T220952, start_date=20240921T221229, end_date=20240921T221229
[2024-09-21T22:12:29.742+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 416 for task trigger_stage_processing (unsupported operand type(s) for +: 'int' and 'NoneType'; 1888)
[2024-09-21T22:12:29.785+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-09-21T22:12:29.822+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
